puphpet/config.yaml.template

```diff
@@ -60,10 +71,20 @@ vagrantfile-local:
         host: detect
 server:
     install: '1'
+    apt-ppa-repos:
+        - ppa:git-core/ppa   # Provides newer git-core. -bp
+        - ppa:chris-lea/node.js  # Provides newer node and npm. -bp
     packages:
-        - vim
-        - memcached
         - curl
+        - emacs
```

Comment:
Sorry about that, the comments didn't show in the Github diff. Should we consider commenting out packages that we know aren't needed in every project then? It would be much easier to hunt for them then and we're not waisting time each time a dev builds the box installing packages no one will ever use .